.flowconfig

```diff
@@ -1,4 +1,5 @@
 [ignore]
+.*/node_modules/.*/node_modules/fbjs/.*
```

Comment:
We have to ignore this because flow is picking up @providesModule from fbjs, this is how react-native does it.